nevergrad/functions/mixsimulator/core.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Based on https://github.com/Foloso/MixSimulator/tree/nevergrad_experiment
+
+import numpy as np
+from ..base import ExperimentFunction
+from mixsimulator.MixSimulator import MixSimulator  #type: ignore
+
+
+class OptimizeMix(ExperimentFunction):
+    """
+        MixSimulator is an application with an optimization model for calculating 
+        and simulating the least cost of an energy mix under certain constraints.
+        
+        For now, it uses a default dataset (more will be added soon).
+        
+        For more information, visit : https://github.com/Foloso/MixSimulator     
+        
+        Parameters
+        ----------
+        time: int
+            total time over which it evaluates the mix (must in hour)
+                
+    """
+    def __init__(self,time: int = 168) -> None:
+        
+        parameters = self.parametrization(time = time)
+        super().__init__(self._simulate_mix , parameters)
+        
+        self.register_initialization()
```

Comment:
```suggestion
        self.register_initialization(time=time)
        self.add_descriptors(time=time)
```

The first line makes sure calling `copy()` on an instance creates a new instance with the same parameters. The second line makes sure the time is stored, for differentiating different cases when plotting results